projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-full-access-view/case-full-access-view.component.html

```diff
@@ -62,46 +59,50 @@ <h2 class="heading-h2 error-summary-heading" id="edit-case-event_error-summary-h
         </mat-tab>
         <mat-tab *ngFor="let tab of sortedTabs; let curIdx=index" [id]="tab.id" [label]="tab.label | rpxTranslate">
           <ng-template matTabContent>
-            <table [class]="tab.id" [attr.aria-label]="'case viewer table' | rpxTranslate">
+            <div class="hide-table-capion" id="table_caption">{{tab.label | rpxTranslate}}</div>
+            <table class="olaaa" [class]="tab.id" aria-labelledby="table_caption">
```

Comment:
is class="olaaa" correct, or should it have a more meaningful value?

Reply:
@anthonydummer It shouldn't be there. It was left after used for testing I will remove now.